ppml/docs/examples.md

```diff
@@ -293,6 +293,93 @@ bash bigdl-ppml-submit.sh \
 ```
 </details>
 
+<details><summary>Trusted SimpleQuery With Multiple KMS/Datasource</summary>
+
+If you have multiple data sources that use different keys, you can also use the `initPPMLContextMultiKMS` method to initialize PPML Context with support for multiple Key Management Systems and data sources.   
+
+You just need to submit the configurations for the KMS and data sources in a manner similar to the following example.
+
+For KMS, you should first submit the number of kms `spark.bigdl.kms.multikms.instance`, and then submit the configurations for each KMS in turn:
+```
+spark.bigdl.kms.multikms.instance=num of your kms
```

Comment:
Please use a natural language sentence to describe the parameter, and pay attention to the text format.
You can refer the format here: https://github.com/intel-analytics/BigDL/tree/main/scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation#parameters